Gems/Atom/RPI/Code/Source/RPI.Public/Pass/ComputePass.cpp

```diff
@@ -58,6 +58,12 @@ namespace AZ
                 return;
             }
 
+            // Hardware Queue Class
+            if (passData->m_useAsyncCompute)
```

Comment:
What if someone wants to do a copy on the copy queue via pass files. Maybe use 
m_graphicsQueue directly in the pass file instead of UseAsyncQueue. So someone can specify a Queue directly in the pass like so

Queue : Graphics 
or
Queue : Compute 
or
Queue : Copy

Reply:
I added this functionality, there is a copy pass and copy pass data, and to the copy pass data I added UseCopyQueue.